src/test/java/uk/gov/hmcts/ccd/domain/service/createcase/SubmitCaseTransactionTest.java

```diff
@@ -240,7 +241,7 @@ void shouldPersistCreateCaseEvent() throws IOException {
             idamUser,
             caseEventDefinition,
             inputCaseDetails,
-            IGNORE_WARNING);
+            IGNORE_WARNING, null);
```

Comment:
Put the `null` parameter on a new line just so it's consistent with all the other parameters.

Reply:
done